nexus/src/app/background/inventory_collection.rs

```diff
@@ -168,45 +161,29 @@ async fn inventory_activate(
 struct DbSledAgentEnumerator<'a> {
     opctx: &'a OpContext,
     datastore: &'a DataStore,
-    page_size: NonZeroU32,
 }
 
 impl<'a> nexus_inventory::SledAgentEnumerator for DbSledAgentEnumerator<'a> {
     fn list_sled_agents(
         &self,
     ) -> BoxFuture<'_, Result<Vec<String>, InventoryError>> {
         async {
-            let mut all_sleds = Vec::new();
```

Comment:
Some time after this code was written, the same behavior was factored into `sled_list_all_batched()`, so I switched to that here.